Atlas/android/app/src/main/java/com/bounswe2017/group10/atlas/adapter/CommentRow.java

```diff
@@ -23,7 +24,17 @@ public String getDate() {
     }
 
     public void setDate(String date) {
+
```

Comment:
The functionality of this function seems to be complex. Can you write unit tests to ensure that it works properly?

Reply:
will be done later